analyzer/src/main/kotlin/curation/Sw360PackageCurationProvider.kt

```diff
@@ -113,3 +117,10 @@ class Sw360PackageCurationProvider(sw360Configuration: Sw360StorageConfiguration
         return SW360ConnectionFactory().newConnection(sw360ClientConfig)
     }
 }
+
+private fun Collection<SW360SparseLicense>.toSpdx(): SpdxExpression? =
+    mapTo(mutableSetOf()) {
+        it.shortName.toSpdx()
+    }.reduceOrNull { left, right ->
+        SpdxCompoundExpression(left, SpdxOperator.AND, right)
+    }
```

Comment:
How about this instead:

    DeclaredLicenseProcessor.process(mapTo(mutableSetOf()) { it.shortName}).spdxExpression